active/0021-transfer-files-over-mqtt.md

```diff
@@ -6,15 +6,15 @@
 
 ## Abstract
 
-This documents defines protocol to send files from MQTT clients to MQTT server. It is using only PUBLISH and PUBACK messages and does not require MQTT 5.0 features like topic aliases.
+This document defines protocol to send files from MQTT clients to MQTT server. It is using only `PUBLISH` and `PUBACK` messages, and requires only the _`PUBACK` Reason Code_ feature which is specific to MQTT 5.0.
```

Comment:
Why `requires`? MQTT < 5 implementations would still work, albeit not as efficient. Or?

Reply:
Right, original wording was more accurate. Backed out last commit.